inginious/frontend/task_dispensers/util.py

```diff
@@ -144,6 +144,9 @@ def get_config(self):
         :return: the config dict of this section
         """
         return self._config
+    @staticmethod
+    def have_accessible_tasks(tasks):
+        return any([task.get_accessible_time().is_open() for task in tasks])
```

Comment:
It's only used at one place. I'd rather put this line there in `inginious/frontend/templates/task_dispensers/toc.html`

Current changes are not better OOP: while it actually concerns a section, it does not have sense to ask for the tasks as they could be contained in the section properties.